src/Linux/Avalonia.LinuxFramebuffer/Output/DrmBindings.cs

```diff
@@ -142,10 +142,21 @@ public unsafe class DrmCard : IDisposable
         public int Fd { get; private set; }
         public DrmCard(string path = null)
         {
-            path = path ?? "/dev/dri/card0";
-            Fd = open(path, 2, 0);
-            if (Fd == -1)
-                throw new Win32Exception("Couldn't open " + path);
+            if (path == null)
+            {
+                Fd = open("/dev/dri/card0", 2, 0);
```

Comment:
While hardcoding card0 as default was somewhat reasonable, I don't think that hardcoding both card0 and card1 is reasonable. I think we should enumerate device files in `/dev/dri/card[0-9]+` and pick the first one sorting by number.

Reply:
Fair enough, hard coding just card0 and card1 does seem quite arbitrary. Although at the same time only checking cards 0 through 9 also feels just as arbitrary.

Perhaps it would be better if the constructor take a param array of strings that it can iterate through, that way developers can choose exactly what cards are checked?

At the moment if you want to check multiple cards you have to catch the Win32Exception, which is pretty clunky, especially if the exception came from a different source.

Reply:
What I understood is that `/dev/dri/card[0-9]+` is a regex expression for all names starting with `card` and any combination of digits after, so not only 0-9.

Additionally, I agree that being able to specify an option for a specific card would be good, as the dev may not always want to use the first one that works, but as fallback it's great.

Reply:
Woops! My mistake, I knew that was a regex of some sort but I must admit my regex interpretation is a little rusty.

I'll implement a commit that allows for specific card selection, and will fallback to trying every card in the /dev/dri/ directory.